_posts/en/posts/2019-12-11-hidden-service.md

```diff
@@ -0,0 +1,25 @@
+---
+title: bitcoincore.org hidden service
+name: hidden-service
+id: en-hidden-service
+lang: en
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement.  English posts only
+announcement: 0
+
+excerpt: >
+  After frequent requests, this site is now reachable as a Tor hidden service
+---
+After frequent requests, this site is now reachable as a Tor hidden service
+through an onion address:
+
+http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/
```

Comment:
Nit: I think this should be placed in angle braces so that it becomes a clickable link, e.g.:

    <http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/>

Reply:
In my experience those links get opened in the wrong browser, so it's arguably better to keep them unclickable.

Reply:
I'm fine with it either way, but if it's a link, you can right-click on it and select "copy link" (or tap and hold on mobile).  If it's text, you have to highlight it before copying.  I find the right-click method slightly easier on desktop and significantly easier on mobile.

Reply:
Sorry forgot about this a bit.
Good point, I'll update it to be a link.